src/components/FontStager/index.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+    font-family: Roboto500;
+    src:
+            url('~typeface-roboto/files/roboto-latin-500.woff2') format('woff2'),
+            url('~typeface-roboto/files/roboto-latin-500.woff') format('woff');
+    font-weight: 500;
+    font-style: normal;
+}
+@font-face {
+    font-family: Roboto300;
+    src:
+            url('~typeface-roboto/files/roboto-latin-300.woff2') format('woff2'),
+            url('~typeface-roboto/files/roboto-latin-300.woff') format('woff');
+    font-weight: 300;
+    font-style: normal;
+}
+@font-face {
+    font-family: Roboto700;
+    src:
+            url('~typeface-roboto/files/roboto-latin-700.woff2') format('woff2'),
+            url('~typeface-roboto/files/roboto-latin-700.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+}
+
+/*
+  The purpose of defining class stages is to
+  re-render once a stage has been met. We start
+  with the minimal default stage of sans-serif,
+  and progressively re-render.
+*/
+html, body {
+    font-family: sans-serif;
+    font-weight: 500;
+    -webkit-font-smoothing: antialiased;
+}
+
+/*
+  The defined stages now modify the display of
+  elements once they are loaded.
+*/
+
+/*
+  During primary stage we only load the Roboto font.
+  Once it's loaded, update the body to use it.
+*/
+.font-stage-primary html,
+.font-stage-primary body {
+    font-family: Roboto500, sans-serif;
+}
+
+/* Prevent the secondary fonts from being tree-shaken away */
+.font-stage-secondary .roboto300 {
+    font-family: Roboto300, sans-serif;
+}
+.font-stage-secondary .roboto700 {
+    font-family: Roboto700, sans-serif;
+}
```

Comment:
According to the [docs](https://material-ui.com/style/typography/#roboto-font-cdn), Material-UI default typography configuration only relies on 300, 400 and 500 font weights. This file is using weight 700. I think we can just use https://github.com/taskcluster/taskcluster-web/blob/a7cc659411351296a42243fa835ae9ae9164c99b/src/components/FontStager/index.css.